lib/phoenix_component.ex

```diff
@@ -8,43 +8,55 @@ defmodule Phoenix.Component do
 
   Here is an example:
 
-      defmodule MyComponent do
-        use Phoenix.Component
+  ```elixir
```

Comment:
Sorry for the nitpick, but can you please keep the Elixir examples four spaces indented instead of using triple backticks?

Reply:
Sure, I'll revert these changes 🙂 

I am curious though if you'd be interested to move towards explicit code blocks for Markdown. Although it is more verbose, it allows tree-sitter and other Markdown parsers to properly handle language injection for accurate syntax highlighting.

Reply:
In this case, I'm with José on the default 4 space indention being elixir, and ex_doc handles it properly so no need for the extra noise :)